runtime/compiler/control/HookedByTheJit.cpp

```diff
@@ -7098,12 +7103,17 @@ int32_t setUpHooks(J9JavaVM * javaVM, J9JITConfig * jitConfig, TR_FrontEnd * vm)
    
    if (compInfo->getPersistentInfo()->getJITaaSMode() == SERVER_MODE)
       {
-      TR_Listener *listener = ((TR_JitPrivateConfig*)(jitConfig->privateConfig))->listener; 
+      int32_t statisticsFrequency = TR::Options::getStatisticsFrequency();
+      TR_Listener *listener = ((TR_JitPrivateConfig*)(jitConfig->privateConfig))->listener;
+
       listener->startListenerThread(javaVM);
 
       if (TR::Options::getVerboseOption(TR_VerboseJITaaS))
          TR_VerboseLog::writeLineLocked(TR_Vlog_JITaaS, "Started JITaaSServer listener thread: %p ", listener->getListenerThread());
 
+      TR_StatisticsThread *statisticsThread = ((TR_JitPrivateConfig*)(jitConfig->privateConfig))->statisticsThread;
+      statisticsThread->startStatisticsThread(javaVM, statisticsFrequency);
```

Comment:
The statisticsThread could get the frequency by itself since that is a static global. No need to pass it as a parameter.